README.md

```diff
@@ -19,6 +19,7 @@ Spindexer works by:
  - Postgres running
  - postgresql-client installed
  - ts-node installed globally
+ - ipfs installed
```

Comment:
i guess since we now take in the IPFS_NODE_HTTP_API, having it locally installed is not necessary anymore, so we can remove this?

```suggestion
```